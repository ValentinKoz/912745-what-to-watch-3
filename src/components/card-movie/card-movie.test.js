import React from "react";
import renderer from "react-test-renderer";
import CardMovie from "./card-movie.jsx";

const film = {
  genre: ``,
  title: `Booksmart`,
  releaseDate: 0,
  poster: `img/fantastic-beasts-the-crimes-of-grindelwald.jpg`,
  bgPoster: ``,
  ratingScore: ``,
  ratingLevel: ``,
  ratingCount: 1000,
  text: ``,
  starring: ``,
};

it(`CardMovie is renderer correctly`, () => {
  const tree = renderer.create(
      <CardMovie film={film} handleEventHover={() => {}} onCardClickHandle={() => {}}/>
  ).toJSON();

  expect(tree).toMatchSnapshot();
});
