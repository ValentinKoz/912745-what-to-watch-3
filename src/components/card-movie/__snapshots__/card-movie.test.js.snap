// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CardMovie is renderer correctly 1`] = `
<article
  className="small-movie-card catalog__movies-card"
  onMouseOver={[Function]}
>
  <div
    className="small-movie-card__image"
    onClick={[Function]}
  >
    <img
      alt="Booksmart"
      height="175"
      src="img/fantastic-beasts-the-crimes-of-grindelwald.jpg"
      width="280"
    />
  </div>
  <h3
    className="small-movie-card__title"
  >
    <a
      className="small-movie-card__link"
      href="#"
      onClick={[Function]}
    >
      Booksmart
    </a>
  </h3>
</article>
`;
